src/navigation/components/Navigation/Navigation.tsx

```diff
@@ -43,6 +49,8 @@ export const Navigation: React.FC<NavigationProps> = ({data, logo, className}) =
     const onSidebarOpenedChange = (isOpen: boolean) => setIsSidebarOpened(isOpen);
 
     useEffect(() => {
+        if (!withBorderOnScroll) return () => {};
+
         const showBorderOnScroll = () => {
             if (!showBorder) {
```

Comment:
Change `!showBorder -> !withBorder` in this place, and if `window.scrollY === 0`, the border will disappear.
it is what you want ?)